.eslintrc.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I like to see that we are bringing more standards into this code repo, but can't help but to wonder If changes like these could have been made in a separate pull request.

In my everyday 9-5 I tried to avoid creating pull requests that exceed the modification of 10 files. (35 files in this PR). Reviewing 35 files at once comes quite painful for your colleagues.

Reply:
Essentially a PR could represent the smallest working modification one could make.

e.g. 

- PR 1: Update editor configs and aslant setup for better code standards
- PR2: Remove private API key from client side app's version control
- PR3: Setup server for communicating with the external movie DB API
- PR4: Get the CategoriesMoviesPreview from our own server
- PR4: Get the RelatedMoviesByID from our own server
- ...

Reply:
This might help you understand how to work with changes that depend on each other with git:
https://blog.logrocket.com/using-stacked-pull-requests-in-github/#:~:text=To%20stack%20two%20PRs%2C%20checkout,develop%20)%20and%20push%20your%20changes.&text=Create%20the%20PR%20with%20the,second%20branch%20from%20the%20first.&text=This%20effectively%20turns%20ts%2Dsetup,ready%20to%20become%20stacked%20PRs.